doc/api/util.md

```diff
@@ -865,6 +865,96 @@ added: v8.0.0
 * {symbol} that can be used to declare custom promisified variants of functions,
 see [Custom promisified functions][].
 
+## util.report.getReport([err])
+<!-- YAML
+added: REPLACEME
+-->
+
+* `err` {Object}
+* Returns: {Object} Returns the diagnostics report as an Object
+
+Generates a JSON-formatted diagnostic report summary of the running process.
+The report includes JavaScript and native stack traces, heap statistics,
+platform information, resource usage etc.
+
+```js
+const { report } = require('util');
+const data = report.getReport();
+console.log(data);
+```
+
+Additional documentation on diagnostic report is available
+at [report documentation][]
+
+## util.report.setDiagnosticReportOptions([options]);
+<!-- YAML
+added: REPLACEME
+-->
+
+Set the runtime configuration of diagnostic report data capture. Upon invocation
+of this function, the runtime is reconfigured to generate report based on
+the new input.
+
+* `options` {Object}
+  * `events` {string[]}
+    * `signal`: generate a report in response to a signal raised on the process.
+    * `exception`: generate a report on unhandled exceptions.
+    * `fatalerror`: generate a report on internal fault.
+      (such as out of memory errors or native assertions)
+  * `signal` {string} sets or resets the signal for report generation
+    (not supported on windows). Default signal is `SIGUSR2`.
+  * `filename` {string} name of the file to which the report will be written.
+  * `path` {string} location at which the report will be generated.
+  * `verbose` {boolean} flag that controls additional verbose information on
+    report generation.
+
+```js
+const { report } = require('util');
+
+// Trigger a report upon uncaught exceptions or fatal erros.
+report.setDiagnosticReportOptions({ events: ['exception', 'fatalerror'] });
+
+// Change the default path and filename of the report.
+report.setDiagnosticReportOptions({ filename: 'foo.json', path: '/home' });
+```
+
+Signal based report generation is not supported on Windows.
+
+Additional documentation on diagnostic report is available
+at [report documentation][]
+
+## util.report.triggerReport([filename, err])
+<!-- YAML
+added: REPLACEME
+-->
+
+* `filename` {string} The file to write into. The `filename` should be
+a relative path, that will be appended to the specified directory, or
```

Comment:
`to the specified directory` -> `to the directory specified by util.report.setDiagnosticReportOptions` ?

Reply:
fixed